src/Stripe.net/Services/_common/RawRequestOptions.cs

```diff
@@ -12,5 +12,27 @@ public class RawRequestOptions : RequestOptions
 
         /// <summary>Gets or sets the value or Stripe-Context request header.</summary>
         public string StripeContext { get; set; }
+
+        internal new RawRequestOptions Clone()
+        {
+            return (RawRequestOptions)this.MemberwiseClone();
+        }
+    }
+
+    #pragma warning disable SA1402 // FileMayOnlyContainASingleType
+    internal static class RawRequestOptionsExtensions
```

Comment:
Eventually, but I don't see a way to define it on `RequestOptions` and have it work correctly for `RawRequestOptions` and if we added the `RequestOptions` version now, it would be dead code.

Reply:
Let's make sure to remove  RawRequestOptionsExtensions when we add a general version then.

Reply:
I learned a little more about C# and did [make it generic](https://github.com/stripe/stripe-dotnet/pull/2830/commits/bc8a20156c0024b693babe0441d10bfe5c81440d). The `new()` constraint is pretty neat.